### ISSUE TITLE
problem launching pac2asbru.pl migration script

### ISSUE THREAD

**Issue by efferre79** [asbru-cm/asbru-cm#994]
I have just installed asbru-cm 6.4.0 but I get a strange error at startup from CLI:

```
$ asbru-cm
INFO: Ásbrú Connection Manager 6.4.0 (asbru-cm) started with PID 6037
INFO: Language detected : it_IT.UTF-8, forcing to en_US.UTF-8
Can't exec "'/usr/bin/perl' /usr/share/asbru-cm/utils/pac2asbru.pl": No such file or directory at /usr/bin/asbru-cm line 237.
INFO: Migration aborted or errors occured, execute with --verbose to see more information (-1)
```
As you can see I have changed the installation path of the scripts. If I change the following line

https://github.com/asbru-cm/asbru-cm/blob/5275d9c84bc0a37c8b3c5dfdbf6704bd4b781ed9/asbru-cm#L237

to

`if ($status = system("'$^X'", "$PATH/utils/pac2asbru.pl", $old_dir, $new_dir, $VERBOSE)) {`

then I get the error:

```
INFO: Ásbrú Connection Manager 6.4.0 (asbru-cm) started with PID 6497
INFO: Language detected : it_IT.UTF-8, forcing to en_US.UTF-8
Can't exec "'/usr/bin/perl'": No such file or directory at /usr/bin/asbru-cm line 237.
INFO: Migration aborted or errors occured, execute with --verbose to see more information (-1)
```

but if I change finally to:

`if ($status = system("$^X", "$PATH/utils/pac2asbru.pl", $old_dir, $new_dir, $VERBOSE)) {`

then it works as expected. I am working on Gentoo Linux. My perl knowledge is limited so I cannot understand what is happening. I have noticed that the explicit perl exec name in the `system()` call has been added [here](https://github.com/asbru-cm/asbru-cm/commit/2c2bba13f7327012e0d2e3abdef311ec917757f1), without `"$^X"` is also working fine on my system.

**Comment by gfrenoy** [comment]
Thanks for the feedback!  In perl, system takes either a full command either the command and a list of parameters, I overlooked this in the code review.  This will be fixed in the next release.

**Comment by gfrenoy** [comment]
A fix is available in our [snapshot](https://docs.asbru-cm.net/General/Installation/) repository.  Let us know if you have the chance to test this and feel free to reopen if it should not work as expected.

In any case, it will be included in our next release (6.4.1).

**Comment by efferre79** [comment]
tested and works, thanks!
